packages/svelte2tsx/svelte-jsx.d.ts

```diff
@@ -700,6 +663,25 @@
       scrollX?: Window['scrollX'];
       scrollY?: Window['scrollY'];
       readonly online?: Window['navigator']['onLine'];
+
+
+      onbeforeprint?: EventHandler<Window>;
```

Comment:
I think all `EventHandler<Window>` should be `EventHandler<Event, Window>` instead.

Reply:
Yeah, that's what I means about the problem with generic parameter

Reply:
Fixed